arm-documentdb/2015-04-08/swagger/documentdb.json

```diff
@@ -0,0 +1,776 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DocumentDB",
+    "description": "Azure DocumentDB Database Service Resource Provider REST API",
+    "version": "2015-04-08"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "Impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}": {
+      "get": {
+        "operationId": "DatabaseAccounts_Get",
+        "description": "Retrieves the properties of an existing Azure DocumentDB database account.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The database account properties were retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/DatabaseAccount"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "DatabaseAccounts_Patch",
+        "description": "Patches the properties of an existing Azure DocumentDB database account.",
+        "x-ms-long-running-operation": true,
```

Comment:
My assumption is that, given the lack of an expected 201, the service does not support upsert (further implied by the lack of otherwise required parameters for create from the body parameter). 

Still need an answer to if a 202 is supposed to be modeled here...